requirements.txt

```diff
@@ -1,6 +1,11 @@
-hepunits
-numpy>=1.11.0
-pandas<0.21; python_version=="3.4.*"
-pandas; python_version == "2.7.*"
-pandas; python_version >= "3.5"
+hepunits==1.0.*
+particle==0.6.*
+decaylanguage==0.5.*
+awkward>=0.12.0,<1.0
+uproot-methods>=0.7.0,<1.0
+uproot>=3.10.0,<4.0
+boost-histogram==0.5.*
+iminuit==1.3.*
+numpy>=1.13.1
+pandas
```

Comment:
I don't think we should require pandas. That's not one of our packages and a user choice. It also may start breaking Python 2 since they've dropped Python 2 support and don't want to touch it. Numpy has dropped Python 2 also, but I think they will at least fix bugs if needed.

Reply:
True, it was needed before for the bayesian_block algorithm which is now moved to `scikit-stats`. I didn't see the harm of having pandas in the requirements since it's pretty standard. About dropping python 2, shouldn't we do it as well?

Reply:
If a requirement has it, then it gets added through that - we are not adding all the other requirement requirements here. Numpy is nice just because we make the minimum version clear. Pandas already breaks wheel building using Python 2, so adding it is a bit heavy.

We need to decide on a time and date to drop Python 2, then do it across (most of) Scikit-HEP - and that hasn't been discussed or planned yet. We could also do the hybrid drop method - each new package supports Python 2 for it's first non-beta release, then is allowed to drop it.

Plans for dropping Python 2 need to be discussed elsewhere, though.

Reply:
Yes, plans for dropping Python 2 should be done elsewhere and including all package admins. I did not even wanted to mention it here ;-).